src/main/webapp/i18n/en/subject.json

```diff
@@ -65,7 +65,8 @@
             "showTokenUrl" : "Show token data",
             "hideTokenUrl" : "Hide token data",
             "baseUrl" : "Base Url",
-            "tokenName" : "Token Name"
+            "tokenName" : "Token Name",
+            "tokenTimeoutMessage": "Please note that this QR code content will expire in {{time}} hour(s). <br>Please refresh to renew the QR code."
```

Comment:
Instead (or in addition) to the time-out, it would be good to show the time of the user, when it will time-out. For US that would be Dec 5, 20:17 PM, for NL 5 dec, 20:17 uur.